monkey/monkey_island/Pipfile

```diff
@@ -5,23 +5,18 @@ name = "pypi"
 
 [packages]
 pyinstaller = "==3.6"
-awscli = "==1.20.44"
 bcrypt = "==3.2.0"
 boto3 = "==1.18.44"
 botocore = "==1.21.44"
-cffi = ">=1.8,!=1.11.3"
```

Comment:
See comment in agent pipfile

Reply:
Not sure why it's not 1.11.3, but it's resolving to 1.15.0 and since it's not a direct dependency I think it's GTG